city_scrapers/spiders/il_capital_development.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlCapitalDevelopmentSpider(CityScrapersSpider):
+    name = "il_capital_development"
+    agency = "Illinois Capital Development Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www2.illinois.gov/cdb/about/boardmeetings"]
+    location = {
+        "name": "James R. Thompson Center",
+        "address": "100 West Randolph Street, 14th Floor, Chicago, IL 60601"
+    }
+
+    def parse(self, response):
+        """
+        """
+        for item in response.xpath("//tbody/tr"):
+            self._validate_location(item)
+            meeting = Meeting(
+                title="CDB Board Meeting",
```

Comment:
We try to leave "Meeting" out of titles we're populating statically, so "Capital Development Board" might work better here